src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -416,7 +416,7 @@ void CoilCoolingDX::oneTimeInit(EnergyPlusData &state)
     SetupOutputVariable(state,
                         "Cooling Coil Dehumidification Mode",
                         Constant::Units::None,
-                        this->dehumidificationMode,
+                        (int &)this->dehumidificationMode,
```

Comment:
Have to do this upcast (it's safe) for doing a `SetupOutputVariable` on enumerations.